packages/contracts-bedrock/src/L1/OptimismPortal2.sol

```diff
@@ -122,8 +123,9 @@ contract OptimismPortal2 is Initializable, ResourceMetering, ReinitializableBase
     /// @notice Address of the ETHLockbox contract.
     IETHLockbox public ethLockbox;
```

Comment:
We should add a comment here explaining that in the current version of the portal, it's impossible to set this value, and it's only possible to have this set if you came from v4.0.0, and chains that skipped that and went straight to 4.1.0 won't have this set